centreon/tests/e2e/cypress/e2e/Authentication/Web-SSO/03-web-sso-authentication-only.feature

```diff
@@ -1,9 +1,10 @@
+@REQ_MON-22140
 Feature: Web SSO authentication
     As an admin of Centreon Platform
     I want to be able to use an external authentication provider
     So platform users can use an existing authentication services to authenticate
 
-@ignore-scenario
+@TEST_MON-22143-scenario
```

Comment:
should be ignored ?

Reply:
@kduret my bad on this, i tried to clean all the `@ignore` that i used for testing, so i replaced all the `@ignore` with `''`.